### DISCUSSION TITLE
classes/class_animationnodeblendspace2d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#292] [User-contributed notes]
# classes/class_animationnodeblendspace2d

Inherits: AnimationRootNode< AnimationNode< Resource< RefCounted< Object A set of AnimationRootNode s placed on 2D coordinates, crossfading between the three adjacent ones. Used by AnimationTree. D...

https://docs.godotengine.org/en/stable/classes/class_animationnodeblendspace2d.html

**Comment by steveweenie** [comment]
How do I go about synchronizing this across multiplayer clients?
